content/blog/_posts/inclusion-good-for-business.md

```diff
@@ -0,0 +1,47 @@
+
+The recent _[Fortune/Deloitte CEO Study](https://www2.deloitte.com/us/en/pages/chief-executive-officer/articles/ceo-survey-gauging-priorities.html)_ found that 96 percent of CEOs now consider diversity, equity, and inclusion (that’s the DEI you’re starting to see more often in the corporate vernacular) to be a strategic priority, integral to building a culture that’s better reflective of the world in which we live. And it acknowledges and embraces the uniqueness that every single person brings to a company’s vision. Together, we are better, and that’s better for business.
+
+
+## Watch your language
+
+One way to do this from the outset is to recognize the [importance of language](https://www.bmc.com/blogs/2021-remote-work-considerations/) and how we use it in speech and the written word. As the world has changed, and awareness along with it, once-standard terminology is being retired and replaced with more inclusive language. So, what do we mean by that?
+
+Well, for the tech industry in particular, think of its historical origins—it was created by white men and their conditioning, so terms such as slave, master, whitelist, blacklist, man hours, foreman, etc., were added to the lexicon without a second thought. This language is micro-aggressive—subtle, intentional or unintentional, and containing some sort of bias towards historically marginalized groups. Business leaders are working to eliminate it and introduce new standard terms.
+
+The [Inclusive Naming Initiative](https://inclusivenaming.org/) is a business collective tasked with defining processes and tools to remove harmful language and replace it with neutral terms. BMC is [proud](https://www.bmc.com/corporate/corporate-responsibility-impact.html) to be one of its member organizations.
+
+DEI initiatives also strive to address issues of gender, gender identity, and gender discrimination, and there are new rules for that, too. For example, new social norms around gender identity have created an awareness and ownership of pronouns. It’s rapidly becoming a part of the social discourse, and social media, to include mention of “he/him,” “she/her,” or “they/them” in how you identify and wish to be referred to in the third person.
+
+[Here](https://s7280.pcdn.co/wp-content/uploads/2021/02/Inclusive-language-list.pdf) is a basic guide to problematic terms you might use every day without being conscious of their impact, and alternative options. BMC is addressing this with our own employees and partners through a style guide [resource](https://s7280.pcdn.co/wp-content/uploads/2021/02/Language-compressed-3.pdf) on how to use inclusive language.
```

Comment:
~[Here](https://s7280.pcdn.co/wp-content/uploads/2021/02/Inclusive-language-list.pdf)~ [Here] (https://inclusivenaming.org/language/word-list) is a basic guide published by the Inclusive Naming Initiative listing some problematic terms you might use every day without being conscious of their impact, and alternative options. BMC is addressing this with our own employees and partners through a style guide [resource](https://s7280.pcdn.co/wp-content/uploads/2021/02/Language-compressed-3.pdf) on how to use inclusive language.